### PR TITLE
Add support for redirectUrl prop in place of referrer

### PR DESCRIPTION
**Pull request by oliverturner**
When a user's choice has been saved we display an onward link. This change allows us to make it point back to the page from which they came.

Additionally, this PR moves the confirmation message to the area below the form controls to make the progression clearer to users.

Screenshot 2021-12-14 at 10 56 19

### CODE REVIEW

File: components/x-privacy-manager/package.json [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
     "@financial-times/x-rollup": "file:../../packages/x-rollup",
     "@financial-times/x-test-utils": "file:../../packages/x-test-utils",
-    "bower": "^1.8.8",
-    "check-engine": "^1.10.1",
-    "fetch-mock-jest": "^1.3.0",
-    "sass": "^1.26.5"
+    "bower": "1.8.13",
+    "check-engine": "1.10.1",
+    "fetch-mock-jest": "1.5.1",
+    "sass": "1.44.0"
```
Comment (apaleslimghost) [inline components/x-privacy-manager/package.json:38]: **question**:  were these pinned deliberately? our new Renovate config doesn't pin things any more, so i think it'll be annoyed about this

**Review by apaleslimghost — APPROVED** [review]
looks good!

**suggestion**: as it is, this is a breaking change, which would require a major release of every x-dash component. that's fine! but keeping it backwards compatible and releasing as a minor version would be a little less hassle :)

File: components/x-privacy-manager/package.json (hunk 1 above)
Comment (oliverturner) [reply components/x-privacy-manager/package.json:38]: Not exactly deliberately: this made me realise that my ~/.npmrc file has `save-exact=true`

**Comment by oliverturner** [comment]
So, on the matter of the breaking nature of this change... it's a feature that has never actually been implemented in either of the 2 locations the component is used... so squinting a bit one _might_ conclude that it's safe to release as a minor version?

**Comment by apaleslimghost** [comment]
if a breaking change falls in the woods...

**Comment by apaleslimghost** [comment]
go ahead and release it as minor 😁

**Review by apaleslimghost — APPROVED** [review]

**Pull request merged**